Libplanet/Net/NetMQTransport.cs

```diff
@@ -491,6 +491,8 @@ public async Task SendMessageAsync(BoundPeer peer, Message message)
                 );
                 var tcs = new TaskCompletionSource<IEnumerable<Message>>();
                 Interlocked.Increment(ref _requestCount);
+
+                // FIXME should we also cacel tcs sender side too?
```

Comment:
It seems a typo mistake.

```suggestion
                // FIXME should we also cancel tcs sender side too?
```

Reply:
Right, I've fixed. 👍